tensorflow_addons/losses/python/metric_learning.py

```diff
@@ -72,3 +74,30 @@ def pairwise_distance(feature, squared=False):
     pairwise_distances = math_ops.multiply(pairwise_distances,
                                            mask_offdiagonals)
     return pairwise_distances
+
+@tf.function
+def pairwise_distance_np(feature, squared=False):
```

Comment:
Don't move it here. I prefer to copy `pairwise_distance_np` function in your lifted_test.py. In the future, we should remove it and modify our test cases to check explicit results directly.

Reply:
Okay done